services/common/src/components/common/CoreTableCommonColumns.tsx

```diff
@@ -87,15 +87,24 @@ export interface ITableAction {
   icon?: ReactNode;
 }
 
-export const renderActionsColumn = (
-  actions: ITableAction[],
-  recordActionsFilter?: (record, actions) => ITableAction[],
+export const renderActionsColumn = ({
```

Comment:
I was playing with implementing a fixed action column. I didn't end up enabling for the report table yet, but I ended up converting this to an object prop since it was getting a bit long to pass in a bunch of null values just to get to the last one.

Reply:
Good solution!